sql/bis/_init.sql

```diff
@@ -0,0 +1,217 @@
+/*
+DESCRIPTION:
+	Initial field mapping and prelimilary data cleaning for BIS job applications data
+
+INPUTS: 
+	dob_jobapplications
+
+OUTPUTS:
+	_INIT_BIS_devdb (
+		uid text,
+		job_number text,
+		job_type text,
+		job_desc text,
+		_occ_initial text,
+		_occ_proposed text,
+		stories_init numeric,
+		stories_prop text,
+		zoningsft_init numeric,
+		zoningsft_prop numeric,
+		classa_init numeric,
+		classa_prop numeric,
+		_job_status text,
+		date_lastupdt text,
+		date_filed text,
+		date_statusd text,
+		date_statusp text,
+		date_statusr text,
+		date_statusx text,
+		zoningdist1 text,
+		zoningdist2 text,
+		zoningdist3 text,
+		specialdist1 text,
+		specialdist2 text,
+		landmark text,
+		ownership text,
+		owner_name text,
+		owner_biznm text,
+		owner_address text,
+		owner_zipcode text,
+		owner_phone text,
+		height_init text,
+		height_prop text,
+		constructnsf text,
+		enlargement text,
+		enlargementsf text,
+		costestimate text,
+		loftboardcert text,
+		edesignation text,
+		curbcut text,
+		tracthomes text,
+		address_numbr text,
+		address_street text,
+		address text,
+		bin text,
+		bbl text,
+		boro text,
+		x_withdrawal text,
+		existingzoningsqft text,
+		proposedzoningsqft text,
+		buildingclass text,
+		otherdesc text
+	)
+*/
+
+DROP TABLE IF EXISTS _INIT_BIS_devdb;
```

Comment:
might be useful to seperate the two via schema. 
e.g. `bis._INIT_devdb` vs `now._INIT_devdb`

Reply:
At this point _INIT is the only table separated by datasource. Are schema useful if there are only one table in each?

Reply:
ahhh, you are right, ignore this commend